### PR TITLE
[DOC]: Add warning about rolling sums with large values

### LINKED ISSUE #11645: Bug when computing rolling_mean with extreme value
Hello,
Please consider the following code :

```
import pandas as pd
import numpy as ny

dates = pd.date_range("2015-01-01", periods=10, freq="D")
ts = pd.TimeSeries(data=range(10), index=dates, dtype=ny.float64)
ts_mean = pd.rolling_mean(ts, 5)
print(ts) 
2015-01-01    0
2015-01-02    1
2015-01-03    2
2015-01-04    3
2015-01-05    4
2015-01-06    5
2015-01-07    6
2015-01-08    7
2015-01-09    8
2015-01-10    9
Freq: D, dtype: float64

print(ts_mean)
2015-01-01   NaN
2015-01-02   NaN
2015-01-03   NaN
2015-01-04   NaN
2015-01-05     2
2015-01-06     3
2015-01-07     4
2015-01-08     5
2015-01-09     6
2015-01-10     7
Freq: D, dtype: float64
```

For the last date (2015-01-10), you should obtain 7, which corresponds to [5, 6, 7, 8, 9] mean value.
Now, please replace the 2015-01-03 value by -9+33 extreme value.

```
dates = pd.date_range("2015-01-01", periods=10, freq="D")
ts = pd.TimeSeries(data=range(10), index=dates, dtype=ny.float64)
ts[2] = -9e+33
print(ts)
2015-01-01    0.000000e+00
2015-01-02    1.000000e+00
2015-01-03   -9.000000e+33
2015-01-04    3.000000e+00
2015-01-05    4.000000e+00
2015-01-06    5.000000e+00
2015-01-07    6.000000e+00
2015-01-08    7.000000e+00
2015-01-09    8.000000e+00
2015-01-10    9.000000e+00
Freq: D, dtype: float64
```

And compute rolling_mean again :

```
ts_mean = pd.rolling_mean(ts, 5)
print(ts_mean)
2015-01-01             NaN
2015-01-02             NaN
2015-01-03             NaN
2015-01-04             NaN
2015-01-05   -1.800000e+33
2015-01-06   -1.800000e+33
2015-01-07   -1.800000e+33
2015-01-08    0.000000e+00
2015-01-09    1.000000e+00
2015-01-10    2.000000e+00
Freq: D, dtype: float64
```

As you can see,  from the 2015-01-08,  computation returns an incorrect result i.e [1, 2, 3] instead of [5, 6, 7]. The extreme value has introduced some perturbations in following date computation.

Best regards,

### LINKED ISSUE #28244: data frame rolling std return wrong result with large elements
#### Code Sample, a copy-pastable example if possible

```python
arr = [9.54e+08, 6.225e-01, np.nan, 0, 1.14, 0]
arr = pd.DataFrame(arr)
print(arr.rolling(5,3).std())
###output
              0
0           NaN
1           NaN
2           NaN
3  5.507922e+08
4  4.770000e+08
5  0.000000e+00
#### Problem description

Expected output.iloc[5,0] = 0.551, as we can see the standeviation of last five elements shouldn't be zero. <br>
But if i change the first element from 9.54e+08 to 9.45e+07, the last standeviation is 0.816 which is still wrong. If i change the first element to 9.45e+5, everything is okay.  <br>
The first element should not affect the result since the window=5, but there seems to be a bug with large elements, and a magnitude of e+08 definitely will not overflow.  <br>
There must be some bugs.


#### Output of ``pd.show_versions()``

<details>

INSTALLED VERSIONS
------------------
commit: None
python: 3.6.1.final.0
python-bits: 64
OS: Darwin
OS-release: 18.6.0
machine: x86_64
processor: i386
byteorder: little
LC_ALL: en_US.UTF-8
LANG: en_US.UTF-8
LOCALE: en_US.UTF-8

pandas: 0.24.2
pytest: 3.0.7
pip: 19.2.1
setuptools: 36.4.0
Cython: 0.25.2
numpy: 1.13.1
scipy: 0.19.1
pyarrow: None
xarray: 0.12.3
IPython: 5.3.0
sphinx: 1.5.6
patsy: 0.4.1
dateutil: 2.6.0
pytz: 2017.2
blosc: None
bottleneck: 1.2.1
tables: 3.4.2
numexpr: 2.6.2
feather: None
matplotlib: 2.0.2
openpyxl: 2.4.7
xlrd: 1.0.0
xlwt: 1.2.0
xlsxwriter: 0.9.6
lxml.etree: 3.7.3
bs4: 4.6.0
</details>

### PR DESCRIPTION
**Pull request by phofl**
- [x] closes #11645 
- [x] closes #28244 (probably closes that one too)
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

Added a warning within the user guide.

### CODE REVIEW

File: doc/source/user_guide/groupby.rst [hunk 1]
```diff
@@ -851,6 +851,13 @@ and that the transformed data contains no NAs.
 Window and resample operations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. warning::
+
+    When using ``rolling()`` and an associated function the results are calculated with rolling sums. As a consequence
```
Comment (mroeschke) [inline doc/source/user_guide/groupby.rst:856]: I think this warning is more appropriate in `computation.rst` where a lot of the window documentation is

Comment (phofl) [reply doc/source/user_guide/groupby.rst]: Yes that is better, thanks

File: doc/source/user_guide/computation.rst [hunk 2]
```diff
@@ -229,6 +229,13 @@ see the :ref:`groupby docs <groupby.transform.window_resample>`.
 
    The API for window statistics is quite similar to the way one works with ``GroupBy`` objects, see the documentation :ref:`here <groupby>`.
 
+.. warning::
+
+    When using ``rolling()`` and an associated function the results are calculated with rolling sums. As a consequence
+    when having values differing with magnitude :math:`1/np.finfo(np.double).eps` this results in truncation. It must be
```
Comment (jreback) [inline doc/source/user_guide/computation.rst:235]: also might be worth noting that we use kahan summation (add a wiki leak) to preserve accuracy as much as possible)

Comment (phofl) [reply doc/source/user_guide/computation.rst:235]: Done

**Review by mroeschke — APPROVED** [review]

**Comment by jreback** [comment]
thanks @phofl very nice! keep em coming!

**Pull request merged**